src/plugins/ini/ini.c

```diff
@@ -198,13 +357,11 @@ int elektraIniOpen(Plugin *handle, Key *parentKey ELEKTRA_UNUSED)
 	KeySet *config = elektraPluginGetConfig (handle);
 	IniPluginConfig *pluginConfig = (IniPluginConfig *)elektraMalloc (sizeof (IniPluginConfig));
 	Key *multilineKey = ksLookupByName (config, "/multiline", KDB_O_NONE);
-	Key *autoSectionKey = ksLookupByName(config, "/autosections", KDB_O_NONE);
-	Key *toMetaKey = ksLookupByName(config, "/meta", KDB_O_NONE);
-	Key *preserveKey = ksLookupByName(config, "/preserveorder", KDB_O_NONE);
-	pluginConfig->preserverOrder = preserveKey != 0;	
+	Key *simpleKey = ksLookupByName(config, "/simple", KDB_O_NONE);
```

Comment:
Maybe `nosection`? Or a enum `/section` which is `none`, `null`, or `always`. (`null` means that for null-keys you get a section, which is the current behaviour of ini-plugin)